rust/agents/processor/src/processor.rs

```diff
@@ -216,10 +216,10 @@ impl Replica {
         };
 
         if proof.leaf != message.to_leaf() {
-            bail!(ProcessorError::ConflictingUpdateAttemptError {
+            bail!(ProcessorError::ProverConflictError {
```

Comment:
Sorry this is admittedly unrelated. [Old PR](https://github.com/nomad-xyz/nomad-monorepo-archive/pull/72) accidentally misnamed this error